src/Data/Json/JTable.purs

```diff
@@ -10,41 +10,47 @@ module Data.Json.JTable
 import Prelude
 
 import Data.Argonaut.Core as J
-import Data.Argonaut.JCursor (JsonPrim)
+import Data.Argonaut.JCursor (JsonPrim, runJsonPrim)
 import Data.Array as A
-import Data.Json.JSemantic (toSemanticDef, renderJSemantic)
 import Data.Json.JTable.Internal (JTableQuery(..), JTableOpts) as Reexports
 import Data.Json.JTable.Internal as JT
-import Data.Maybe (Maybe(..))
-import Data.String (joinWith)
+import Data.Maybe (Maybe(..), fromMaybe)
+import Data.String as S
 
 import Halogen as H
 import Halogen.HTML as HH
 import Halogen.HTML.Properties as HP
 
-renderJsonSimple ∷ JsonPrim → String
-renderJsonSimple j = renderJSemantic $ toSemanticDef j
+renderJsonPrim ∷ JsonPrim → String
+renderJsonPrim j = runJsonPrim j (const "") show showNumber id
+  where
+  showNumber n =
+    let s = show n
+    in fromMaybe s (S.stripSuffix (S.Pattern ".0") s)
```

Comment:
Maybe keep the name `renderJsonSimple`? Or `renderJsonDefault` perhaps? Might be useful to export it too.